ui/src/app/richskill/detail/rich-skill-manage/action-bar/manage-rich-skill-action-bar.component.ts

```diff
@@ -127,6 +128,16 @@ export abstract class ManageRichSkillActionBarComponent implements OnInit {
     }
   }
 
+  handleCopyPublicURL(): void {
```

Comment:
We already have an approach for copying URLs (see a duplicated 'onCopyURL' method). If there's a reason that this is superior, I'm open to that. 

Otherwise, us duplicating the existing approach will give us a cleaner path to identifying needed refactors. Thank you.